src/aggregator/aggregator/aggregator.go

```diff
@@ -274,7 +275,8 @@ func (agg *aggregator) AddTimedWithStagedMetadatas(
 	sw := agg.metrics.addTimed.SuccessLatencyStopwatch()
 	agg.updateStagedMetadatas(metas)
 	agg.metrics.timed.Inc(1)
-	shard, err := agg.shardFor(metric.ID)
+	shardingId, _ := parsers.GetMetricIDWithoutLe(metric.ID)
```

Comment:
We need some visibility for this new feature:

1. Let's add a new metric: histrogramCounter{shardId="0"}, that we can verify histogram balance status.
2. let's add a debug log: histogram metric, assigned shardId, pre-aggregation metadata and etc.

Reply:
sounds good, will add that in the next couple commits